spec/helpers/faker/puppet_module_remote_repo.rb

```diff
@@ -0,0 +1,130 @@
+require 'open3'
+
+require_relative '../faker'
+
+# Fake a remote git repository that hold a puppet module
+#
+# This module allow to fake a remote repositiory using:
+#  - a bare repo
+#  - a temporary cloned repo to operate on the remote before exposition to modulesync
```

Comment:
```suggestion
a temporary cloned repo to operate on the remote before exposing to modulesync
```
??? (`exposition` might be correct, but sounds odd to me)

Reply:
I turned it using `exposing`, I think you are more fluent in English than me :-)
Thank you.